public/login.css

```diff
@@ -0,0 +1,58 @@
+html,
+body {
+  height: 100%;
+}
+body {
```

Comment:
Should contain an empty line. Does not linter check this file?

Reply:
If you add this to linter, consider moving styles into subdirectory where you lint everything.

Reply:
It should be processed. Let me check it. 

Btw. I ran into a problem where stylelint doesn't respect ignore comments, i.e. if you use `--fix` option than `sanitize.css` file gets processed and auto fixed. 😞 Maybe sanitize.css should be minimized with webpack or we should have it unminified.

Reply:
It's fine to have `sanitize.js` fixed by linter. We do not have problems with network load there.

By the way, did you changed to local version because of content policy you wrote? Did not you want to add CDN and continue using Sanitize from there?

Reply:
> By the way, did you changed to local version because of content policy you wrote? Did not you want to add CDN and continue using Sanitize from there?


I changed it because of security. It may happen that sanitize.css would contain malicious code. I know it's more than impossible but still, we should be cautious especially when dealing with credentials 😄. If I didn't read the article you sent me I would probably leave it there as it was.

Reply:
https://github.com/oacore/dashboard/pull/140/commits/85d06cb1bd05da4d05942e1030e2975910f8593e